package.json

```diff
@@ -43,9 +49,16 @@
   },
   "devDependencies": {
     "chai": "4.2.0",
-    "chai-http": "4.3.0",
     "chai-as-promised": "7.1.1",
-    "eslint": "7.8.1",
+    "chai-http": "4.3.0",
+    "eslint": "7.2.0",
+    "eslint-config-airbnb": "18.2.0",
```

Comment:
Please use `eslint-config-airbnb-base` to avoid installing unnecessary dependencies.